ui/app/components/list-item-row.js

```diff
@@ -1,10 +1,10 @@
 import Ember from 'ember';
 import hbs from 'htmlbars-inline-precompile';
 
-let LinkedBlockComponent = Ember.Component.extend({
```

Comment:
Any objection to renaming this? We're only using it for these so it seemed clearer

Reply:
I still feel like linked-block is a better description of the component - the only functionality it provides is linking and  and it's not used for rows that aren't links. But that said I'm fine with changing the name too. We generally need to go through and make components for lists, list items, etc.

Reply:
That's true. Maybe it would be better to use `linked-block`, use the `.list-item-row` class for these styles, and then instead of `is-selectable` it would just be the combined `.linked-block.list-item-row`. If that makes sense.

Reply:
I'll leave it up to you! But yeah I think you can add `is-selectable` to the classNames component declaration so you don't have to pass it in everywhere you use the component too.